### PR TITLE
Add ExtractSubContentTypes.

### PR DESCRIPTION
**Pull request by yuizumi**

### CODE REVIEW

File: exchange/response.go [hunk 1]
```diff
@@ -21,6 +21,12 @@ import (
 	"github.com/google/webpackager/resource/preload"
 )
 
+// Keys used in ExtraData.
+const (
+	// See htmltask.ExtractSubContentTypes.
+	SubContentType = "Sub-Content-Type"
```
Comment (tomokinat) [inline exchange/response.go]: optional: Even though this is only for internal use, "Sub-Content-Type" sounds like a general or standard header and can be confusing (TBH I googled once). Prefix Wpkg- or something?

Comment (yuizumi) [reply exchange/response.go]: Done

**Comment by yuizumi** [comment]
PTAL. Changed Godoc comments significantly.

File: exchange/response.go [hunk 2]
```diff
@@ -21,10 +21,11 @@ import (
 	"github.com/google/webpackager/resource/preload"
 )
 
-// Keys used in ExtraData.
+// These are keys used in ExtraData. They are prefixed with "X-WebPackager"
+// to avoid confusion with real HTTP headers.
 const (
 	// See htmltask.ExtractSubContentTypes.
-	SubContentType = "Sub-Content-Type"
+	SubContentType = "X-Webpackager-Sub-Content-Type"
```
Comment (tomokinat) [inline exchange/response.go]: X- prefixed headers are now deprecated.
- https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers
- https://tools.ietf.org/html/rfc6648#section-3

Comment (yuizumi) [reply exchange/response.go]: Removed.

**Comment by tomokinat** [comment]
LGTM

**Pull request merged**